tests/unit_server/arghandler_test.py

```diff
@@ -488,7 +488,7 @@ def test_211__argparse_choices__error(argument_source: str) -> None:
         "pick_it": "paper",
         "bar": "True",
     }
-    choices: list = ["rock", "paper", "scissors"]
+    choices: list[Any] = ["rock", "paper", "scissors"]
```

Comment:
I think just `choices = ["rock", "paper", "scissors"]` is fine for mypy

Reply:
No, it needed `Any` in this case because choices was redefined as a different type of list a few lines later, and mypy got confused.